spec/models/manageiq/providers/azure/cloud_manager/metrics_capture_spec.rb

```diff
@@ -24,28 +24,36 @@
 
     it "returns nothing if the insights service is not registered" do
       allow(ems).to receive(:insights?).and_return(false)
-      expect(metric.perf_collect_metrics('whatever')).to be_nil
+      counters, values = metric.perf_collect_metrics('whatever')
+      expect(counters.blank?).to be_truthy
+      expect(values.blank?).to be_truthy
     end
 
     it "returns nothing if the region is not supported" do
       allow(ems).to receive(:insights?).and_return(true)
       allow(armrest_service).to receive(:rest_get).and_raise(::Azure::Armrest::BadRequestException.new('x', 'y', 'z'))
       expect($log).to receive(:warn).with(/problem collecting metrics/i)
-      expect(metric.perf_collect_metrics('whatever')).to be_nil
+      counters, values = metric.perf_collect_metrics('whatever')
+      expect(counters.blank?).to be_truthy
+      expect(values.blank?).to be_truthy
```

Comment:
might be the best to test we expect the `{}, {}` returned? Or the code works with `nil, nil` too?

Reply:
:+1: done